src/test/java/seedu/recipe/testutil/TypicalRecipes.java

```diff
@@ -228,6 +230,21 @@ public class TypicalRecipes {
                 new Step("Add all Tartar ingredients together and mix. Add salt to taste and hot sauce if you want"
                         + " heat."),
                 new Step("Assemble together and enjoy!"))).build();
+    // Manually added - Person's details found in {@code CommandTestUtil}
+    public static final Recipe CHICKEN = new RecipeBuilder(
+            new Name(VALID_NAME_CHICKEN),
+            RecipePortion.of(VALID_PORTION_CHICKEN),
+            RecipeDuration.of(VALID_DURATION_CHICKEN),
+            Set.of(new Tag(VALID_TAG_CHINESE)),
+            List.of(new Ingredient(VALID_INGREDIENT_CHICKEN)),
+            List.of(new Step(VALID_STEP_CHICKEN))).build();
+    public static final Recipe FISH = new RecipeBuilder(
+            new Name(VALID_NAME_FISH),
+            RecipePortion.of(VALID_PORTION_FISH),
+            RecipeDuration.of(VALID_DURATION_FISH),
+            Set.of(new Tag(VALID_TAG_ITALIAN)),
+            List.of(new Ingredient(VALID_INGREDIENT_FISH)),
```

Comment:
Do verify that your new ingredient names use the new Ingredient string format, and merge `master` from upstream into this branch. Verify merge conflicts with me.